packages/shared/src/lib/rank.ts

```diff
@@ -103,8 +103,8 @@ interface GetNextRankTextProps {
   showNextLevel?: boolean;
 }
 
-export const getRank = (rank: number): number => {
-  const computedRank = Math.max(0, rank - 1);
+export const getRank = (rank: number | undefined): number => {
```

Comment:
Would it not work like this:
```suggestion
export const getRank = (rank = 0): number => {
```

Reply:
I think that only works if the `rank` argument is not passed at all. But in our case, the `rank` is an actual variable that is passed which has the value of `undefined`

Reply:
scratch that, your suggestion works even in the case I mentioned 🤝